frontend/proxies.js

```diff
@@ -89,7 +89,7 @@ function listMethods(context, listId, path) {
     methods[method] = (...args) => {
       const list = context.getObject(listId)
         .map((item, index) => context.getObjectField(path, listId, index))
-      return list[method].call(list, ...args)
+      return list[method].call(list, ...args) // eslint-disable-line
```

Comment:
This is apparently an "unnecessary use of call" but I figured it should get a follow-up patch.